### ISSUE TITLE
Full Support For Biomes O'Plenty

### ISSUE THREAD

**Issue by whitestarx0312** [CyanideX/Unity#156]
There are only a handful of textures for BOP with the majority being left vanilla. I would suggest full support (Mainly woods/leaves) as this is one of the most used mods in modded minecraft and is very visible when exploring worlds.

**Comment by CyanideX** [comment]
The reason I only did a few is because I think BoP is already one of the best looking Minecraft mods out there. You're right though, I could do the logs and leaves at least.

**Comment by whitestarx0312** [comment]
I agree, they are good textures. My suggestion was just for continuity when exploring, and I really like how unity does logs/planks.

**Comment by ate0ate** [comment]
I just wanted to add one minor gripe with BoP support as it stands currently. While this is undoubtedly the single best resource pack ever created for Minecraft, and it is doubtful that the hands of any mere mortal could ever hope to come close to its splendor, there is one very tiny thing that could be improved... the flowering oak leaves.

The standard oak leaves added by Unity are great, as are the flowering oak leaves added by BoP. However, they look a little bit off when they are both together on one tree. The flowering oak trees kinda stick out like a sore thumb with the random scattering of the unity texture mixed along and side by side with the flowering variant from BoP.
